src/components/dashboard/Receive.js

```diff
@@ -69,9 +70,23 @@ const Receive = ({ screenProps, styles, ...props }: ReceiveProps) => {
           </PushButton>
           <View style={styles.space} />
           {isMobile && navigator.share ? (
-            <CustomButton onPress={shareAction}>{SHARE_TEXT}</CustomButton>
+            <CustomButton
+              onPress={() => {
+                fireEvent('RECEIVE', { type: 'wallet' })
```

Comment:
the event should be RECEIVE_DONE

Reply:
why? You've changed the name in comments:
![image](https://user-images.githubusercontent.com/36693645/70513521-7a2c5400-1b3a-11ea-9b02-b3f024c0b476.png)

Reply:
my mistake i ment i wanted the names in the issue after the initial names were incorrect
change to RECEIVE_DONE

Reply:
ok. we will change to RECEIVE_DONE